paper-fab.html

```diff
@@ -103,12 +108,14 @@
       iron-icon {
         @apply(--paper-fab-iron-icon);
       }
-      
+
       span {
         width: 100%;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        text-align: center;
+        @apply(--paper-fab-label);
```

Comment:
Huh, it's fine in Safari/FF/Canary, but borked on my Chrome stable. Hurray. I'll remove the mixin for now, then.

Can I use emoji in my demo???